Makefile

```
@@ -0,0 +1,14 @@
+init:
+	./scripts/init.sh
+
+check:
+	SKIP_WASM_BUILD= cargo check
+
+test:
+	SKIP_WASM_BUILD= cargo test --all
+
+run:
+	WASM_BUILD_TOOLCHAIN=nightly-2020-10-05 cargo run --release -- --dev --tmp
+
+build:
+	WASM_BUILD_TOOLCHAIN=nightly-2020-10-05 cargo build --release
```

```suggestion
.PHONY: init
init:
	./scripts/init.sh

.PHONY: check
check:
	SKIP_WASM_BUILD=1 cargo check

.PHONY: test
test:
	SKIP_WASM_BUILD=1 cargo test --all

.PHONY: run
run:
	WASM_BUILD_TOOLCHAIN=nightly-2020-10-05 cargo run --release -- --dev --tmp

.PHONY: build
build:
	WASM_BUILD_TOOLCHAIN=nightly-2020-10-05 cargo build --release
```

* Those are all [phony targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)
* In my totally personal opinion ending a declaration with a `=` looks like a syntax error.